### DISCUSSION TITLE
LoTW sync successful but many QSOs are not uploaded

### DISCUSSION THREAD

**Discussion by 9a3ty** [magicbug/Cloudlog#3187] [General]
Hi, 

I have started using CloudLog just recently. If I say that I was pleasantly surprised it would be an understatement. 
So, I showed it to my friends in our local radio club and they were very interested in using it, too. To help them I started doing initial setup and upload of logbooks. They did some work too :-) 

I have encountered several problems in this short time and I would like to start solving them.

This time it is LoTW manual sync done only partially.

LoTW said :

2024-09-02 10:16:09 LOTW_QSO: Processing file: 20240902101544.25158
2024-09-02 10:16:09 LOTW_QSO: User file: 9a7jcy-2024-09-02-10-15-44-cloudlog.tq8
2024-09-02 10:16:09 LOTW_QSO: Certificate found for 9A7JCY - CROATIA (497)
2024-09-02 10:16:09 LOTW_QSO: QSO QSLs a contact: 2024-08-28 16:44:31Z 9A7JCY OK7JN/P 40M SSB
2024-09-02 10:16:09 LOTW_QSO: Successfully processed 4 QSO records in 0.433637 seconds
2024-09-02 10:16:09 LOTW_QSO: 1 QSL record entered
2024-09-02 10:16:09 LOTW_QSO: No errors encountered

but cloudlog marked many QSOs as uploaded to LoTW. 
By doing that (I guess) it will not try to upload those QSOs again.

Is there any (easy) way to fix this? 

Thank you in advance for your help.

**Comment by magicbug** [comment]
Quick method is to run SQL query on the LOTW Sent column and set it to "N" 

You say its not uploading some QSOs which ones did it miss and are they outside the date limits of the Lotw certificate?

**Reply by 9a3ty** [reply]
Unfortunately LoTW no longer accepts new uploads with too many duplicates so if I set all LOTW Sent to "N" I will get "LOTW_QSO: Processing ABORTED: Maximum allowed count of duplicate QSOs reached" and whole upload is discarded.

No, QSOs are not outside the date limits of the Lotw certificate. For example QSOs dated 2024-08-28 are uploaded but 2024-08-22 are not but both are now LoTW Sent "Y".
